.github/workflows/build_docker.yml

```diff
@@ -0,0 +1,100 @@
+name: "Build Docker"
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+    paths-ignore:
+      - "**.md"
+  pull_request:
+    branches:
+      - "main"
+    paths-ignore:
+      - "**.md"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      CACHE_IMAGE: "ghcr.io/vllm-project/vllm-spyre:build-cache"
+      CACHE_REGISTRY: "ghcr.io"
+      QUAY_REPOSITORY: "quay.io/ibm-aiu/vllm-spyre"
+
+    permissions:
+      packages: write
+      contents: read
+
+    steps:
+    - name: "Checkout"
+      uses: actions/checkout@v4
+
+    - name: "Set up QEMU"
+      uses: docker/setup-qemu-action@v3
+
+    - name: "Set up Docker Buildx"
+      uses: docker/setup-buildx-action@v3
+
+    - name: "Log in to Github Container registry"
+      uses: docker/login-action@v3
+      if: github.event_name != 'pull_request'
+      with:
+        registry: ghcr.io
```

Comment:
use the `CACHE_REGISTRY` env var

Reply:
🌶️ , went ahead and just removed CACHE_REGISTRY because it wasn't used anywhere else